bencher.krun

```diff
@@ -0,0 +1,29 @@
+execfile("luajit.krun", globals()) # inherit settings from the main experiment
+
+MAIL_TO = ["fsfod11@gmail.com"]
+
+PRE_EXECUTION_CMDS = [
+    "sudo systemctl stop apt-daily.timer",
+    "sudo systemctl --runtime disable apt-daily.timer", 
+    "sudo systemctl stop apt-daily-upgrade.timer",
+    "sudo systemctl --runtime disable apt-daily-upgrade.timer",
+    
+    "sudo systemctl stop systemd-timesyncd",
+    "sudo systemctl --runtime disable systemd-timesyncd",
+    "sudo systemctl stop cron",
+]
+
+POST_EXECUTION_CMDS = [
+    "sudo systemctl start systemd-timesyncd || true",
+    "sudo systemctl start cron || true",
+]
+
+def custom_dmesg_whitelist(defaults):
+    return defaults + [
+      "^.*en[0-9a-zA-Z]+: link (down|up)",
+      "^.*en[0-9a-zA-Z]+: Link is (up|down).*",
+      "^.*en[0-9a-zA-Z]+: Flow control is (on|off).*",
+      "^.*en[0-9a-zA-Z]+: EEE is (enabled|disabled).*",
```

Comment:
We should update the defaults in krun. The old defaults no longer work on debian 9 due to the new interface names in linux:

https://github.com/softdevteam/krun/blob/b022d26a3602b1e2d9fdea48648c7957707baf52/krun/platform.py#L1344-L1349